pacai/student/multiAgents.py

```diff
@@ -111,45 +108,40 @@ def getAction(self, gameState):
 
         gameState.getNumAgents():
             Returns the total number of agents in the game
-        """
+    """
+
+    def __init__(self, index):
+        super().__init__(index)
 
-        # *** Your Code Here ***
-        util.raiseNotDefined()
 
 class AlphaBetaAgent(MultiAgentSearchAgent):
     """
     Your minimax agent with alpha-beta pruning (question 3)
+
+    Method to Implement: 
+
+    - def getAction(self, gameState):
+        - Returns the minimax action using self.treeDepth and self.evaluationFunction
     """
 
     def __init__(self, index):
         super().__init__(index)
 
-    def getAction(self, gameState):
-        """
-        Returns the minimax action using self.treeDepth and self.evaluationFunction
-        """
-
-        # *** Your Code Here ***
-        util.raiseNotDefined()
 
 class ExpectimaxAgent(MultiAgentSearchAgent):
     """
     Your expectimax agent (question 4)
-    """
 
-    def __init__(self, index):
-        super().__init__(index)
-
-    def getAction(self, gameState):
-        """
-        Returns the expectimax action using self.treeDepth and self.evaluationFunction
```

Comment:
Make sure to be consistent with the comments. Add in Method to implement as you did before.